src/Sparrow.Server/Platform/PalHelper.cs

```diff
@@ -7,6 +7,10 @@ namespace Sparrow.Server.Platform
 {
     public static class PalHelper
     {
+        private const int ErrorMediaIsWriteProtectedCode = 19; 
```

Comment:
Can we rename it to `ERROR_WRITE_PROTECT`? We already use such naming convention for error codes e.g. `ExceptionHelper.ERROR_COMMITMENT_LIMIT`